### ISSUE TITLE
Better name?

### ISSUE THREAD

**Issue by francesco-strazzullo** [frameworkless-movement/manifesto#10]
Hi @jevakallio. This issue is related to your issue #8. But I would like to concentrate on the first part of your statement on this specific issue. When you talk about the name of the movement and how is difficult to create a constructive position when it's formulated in the negative.

You made our team think about it. The point is, **you're absolutely right**. 

The purpose of this movement is not just to make people write code without frameworks. But it's also, and perhaps more importantly, on how we as developers make context-aware techinical decisions about frameworks. And to make people understand why it's important to consider a frameworkless approach as a solution.

So a better name could be "Good Context-aware Techincal Decisions about Frameworks" Movement. 😄

But as you can understand "Frameworkless Movement" has more impact, even if it's not exactly what we mean. Now the I hope that I explained better our purpose, I hope that someone can come up with a better name. 

In the meantime, we will change our README in order to make this message clear enough.

Thank you very much to help us making our point as clearest as possible.

**Comment by MattiaFattorello** [comment]
Hi, I like "Frameworkless" Movement as it inspire boldness. Yet, a more concise version of "Good Context-aware Techincal Decisions about Frameworks" could be "Frameworkwise" Movement". :)

**Comment by francesco-strazzullo** [comment]
Thanks @MattiaFattorello for the proposal, we really appreciate it! We decided to keep this discussion open for now, to let people suggest new names to us.

**Comment by giacomocerquone** [comment]
Absolutely, I followed a live about this topic and at first glance I am firmly convinced that the naming is not correct, I initially thought something completely different and a better name could let you reach even more people.

Frameworkwise is more appropriate
